### PR TITLE
Using ES6 syntax and Adding luxon time

### PR DESCRIPTION
**Pull request by shayan1234554321**
Used ES6 Syntax 
-  replaced normal functions with arrow functions

Separated files for accessibility
- Used 3 other files for more readability ( 4rth one is for luxon library and not a file for accessibility )
- Used import and export functionality

Added timing functionality
- Added time in the header
- Added timing styling in CSS
- Used luxon library for getting time
- Used setInterval to update time after each second

### CODE REVIEW

File: scripts/booksToHTML.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+const list = document.getElementById('list');
+/* eslint-disable no-undef */
+const toast = Swal.mixin({
```
Comment (AmaduKamara) [inline scripts/booksToHTML.js:3]: - [ ] Let's try to not disable Linter checks in our codebase as they are there to help us write clean, professional code. Kindly fix the problem so you have a professional codebase with no linter issue 👍🏽

File: scripts/booksToHTML.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  const button = document.createElement('button');
+  button.innerHTML = 'Remove';
+  button.className = 'RemoveButton';
+  button.setAttribute('id', 'removeButton');
+  button.addEventListener('click', ((e) => {
+    const parent = e.target.parentElement;
+    const index = parent.id;
+    remove(index);
+    /* eslint-disable no-use-before-define */
+    addAllBooks(books, remove);
```
Comment (AmaduKamara) [inline scripts/booksToHTML.js]: - [ ] Let's try to not disable Linter checks in our codebase as they are there to help us write clean, professional code. Kindly fix the problem so you have a professional codebase with no linter issue 👍🏽

File: scripts/index.js [hunk 3]
```diff
@@ -1,128 +1,67 @@
...
+
 const form = document.getElementById('form');
 const title = document.getElementById('title');
 const author = document.getElementById('author');
-const list = document.getElementById('list');
+const time = document.getElementById('time');
 
 /* eslint-disable no-undef */
-const toastMixin = Swal.mixin({
+const toast = Swal.mixin({
```
Comment (AmaduKamara) [inline scripts/index.js:16]: - [ ] Let's try to not disable Linter checks in our codebase as they are there to help us write clean, professional code. Kindly fix the problem so you have a professional codebase with no linter issue 👍🏽

File: scripts/index.js [hunk 4]
```diff
@@ -1,128 +1,67 @@
...
   localStorage.books = JSON.stringify(colection.books);
-  toastMixin.fire({
-    animation: true,
+  toast.fire({
     title: 'Book Added',
   });
+};
+
+/* eslint-disable max-len */
+const currentTime = () => DateTime.now().toLocaleString({ ...DateTime.DATETIME_MED_WITH_SECONDS, hour12: true });
```
Comment (AmaduKamara) [inline scripts/index.js]: - [ ] Let's try to not disable Linter checks in our codebase as they are there to help us write clean, professional code. Kindly fix the problem so you have a professional codebase with no linter issue 👍🏽

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️ ♻️ ♻️

Hi @shayan1234554321,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✅ You used Gitflow correctly 👍🏽
✅ Linter checks passed 👍🏽
✅ Used ES6 Modules, arrow function syntax 👍🏽
✅ The app is nicely styled and works fine 👍🏽

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly fix the Linter issues in your codebase so you have a clean and professional codebase. 👍🏽

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: scripts/booksToHTML.js (hunk 1 above)
Comment (shayan1234554321) [reply scripts/booksToHTML.js:3]: I cannot remove this one as I used a library for toast messages and in the documentation, it is how we can use that library. We will import the library in the script in html and then use it like this. So I have to disable the linter for this one but I fixed the others.

File: scripts/index.js (hunk 3 above)
Comment (shayan1234554321) [reply scripts/index.js:16]: I cannot remove this one as I used a library for toast messages and in the documentation, it is how we can use that library. We will import the library in the script in html and then use it like this. So I have to disable the linter for this one but I fixed the others.

**Review by victorokoroji — APPROVED** [review]
Hi @shayan1234554321, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**